core/arch/arm/plat-marvell/conf.mk

```diff
@@ -6,6 +6,13 @@ core_arm32-platform-aflags	+= -mfpu=neon
 ifeq ($(PLATFORM_FLAVOR),armada7k8k)
 include core/arch/arm/cpu/cortex-armv8-0.mk
 platform-debugger-arm := 1
+$(call force,CFG_8250_UART,y)
+endif
+
+ifeq ($(PLATFORM_FLAVOR),armada3700)
+include core/arch/arm/cpu/cortex-armv8-0.mk
+platform-debugger-arm := 1
+$(call force,CFG_MVEBU_UART,y)
 endif
```

Comment:
How about enable `CFG_ARM64_core` and `CFG_ARM_GICV3` here? (since your'e saying in the commit log that those are needed).

Reply:
Sure, that's good.